### ISSUE TITLE
[1.10.2] Build 102 Crash upon landing on Venus after using dimension tp

### ISSUE THREAD

**Issue by SportsFan10** [micdoodle8/Galacticraft#2778]
Galacticraft Version: 1.10.2-4.0.0.102
Crash log: https://pastebin.com/fPmSVenC

I used the dimension tp command to land on Venus and once I land the game freezes, shows the yellow atmosphere of Venus and then crashes.  After loading back in, you are shown floating just above the surface inside the lander before the process of the game freezing again repeats itself.  

Perhaps this is related to: #2748  

Further Testing: Appears to be happening on all Planets and also in survival mode with a standard launch.

**Comment by radfast** [comment]
Thanks for reporting

This is a duplicate of #2775 

This and other recent crash issues will be fixed in build 103, should be available later tonight.  I'll let you know when that's ready.

**Comment by radfast** [comment]
Build 104 now available with this fix.

**Comment by SportsFan10** [comment]
Awesome, thanks for the great work on the mod!  Amazing to see the progress over the years.  Feels like just yesterday the Yogscast showed this amazing mod to the public.  Now I see what it has become and I can't thank you and Micdoodle enough.  Truthfully this is the mod that keeps me interested in Minecraft.

**Comment by radfast** [comment]
:)

We do have a few more additions planned for the mod over the months ahead
